css/03-generic.css

```diff
@@ -220,4 +220,28 @@ h4 {
 /* remove space between blog quotes and featured image */
 .entry-content .wp-block-pullquote:first-child {
 	margin-top: 0;
+}
+
+/* Blockquotes */
+
+blockquote.wp-block-quote {
+	background: var( --color-brandRed );
+	margin: -1.25em;
+	padding: 1.5em !important;
```

Comment:
Is this important required, or is there another, more specific selector you could use to override the padding? Importants breed and can be very bad news over time.

Reply:
I knew you'd catch that. I couldn't figure out a better way to do it. But I am an old man.  @zoracreates could you jump in here and see if you could find a better selector?

Reply:
Also, @zoracreates, I want to add the same polygon effect we did with the headers to these blockquote blocks. Can you take that on?

Reply:
Yes, I can take that on. Is there a ticket for it so I can assign it? Or should I make one?

Reply:
> I knew you'd catch that. I couldn't figure out a better way to do it. But I am an old man. @zoracreates could you jump in here and see if you could find a better selector?

I'll look into it!

Reply:
@zoracreates cool - make sure you pull the latest, I just did this: https://github.com/thec4aa/c4aa-org/pull/77#issuecomment-522169677